src/javascript/app/pages/cashier/cashier.js

```diff
@@ -247,12 +253,36 @@ const Cashier = (() => {
 
                 if (Currency.isCryptocurrency(currency)) {
                     $('.crypto_currency').setVisibility(1);
+                    $('.change-account-btn-crypto').off('click').on('click', (e)=>{
+                        e.stopPropagation();
+                        $('.main-account').trigger('click');
+                        $('html, body').animate({ scrollTop: 0 }, 'slow');
+                    });
 
                     const previous_href = $('#view_payment_methods').attr('href');
                     $('#view_payment_methods').attr('href', previous_href.concat('?anchor=cryptocurrency'));
                 } else {
                     $('.normal_currency').setVisibility(1);
+                    $('.change-account-btn').off('click').on('click', (e)=>{
```

Comment:
it seems like all of these are the same thing and the only thing that changes is the classname here. seems good to make it into a reusable function that accepts the classname as a parameter.